src/common/dice.js

```diff
@@ -133,18 +133,24 @@ class DNDBDice {
         }
         this._rolls = [];
     }
-
+    getDieResult () {
+        // Borrowed from https://pthree.org/2018/06/13/why-the-multiply-and-floor-rng-method-is-biased/
+        const max = Math.floor(2 ** 32 / this.faces) * this.faces; // make "max" a multiple of "range"
+        let x;
+        do {
+            x = Math.floor(Math.random() * 2 ** 32); // pick a number of [0, 2^32).
+        } while (x >= max); // try again if x is too big
+        return x % this.faces; // uniformly picked in [0, range)
```

Comment:
"uniformly picked in [0, range)"
What we want is the range `[1, range]` instead. As it is, rolls are going to become 0-19 instead of 1-20 on a d20.
Also change the comments to say 'faces' instead of 'range' (both here and earlier in this function).